src/main/java/org/svydovets/query/ParameterNameResolver.java

```diff
@@ -58,4 +58,20 @@ public static String resolveColumnName(Field field) {
                 ? field.getName()
                 : columnAnnotation.name();
     }
+
+    /**
+     * This method helps to define list name of declared <strong>entity field</strong> by name from annotation
+     *
+     * @param clazz - entity class annotated as @Id
+     * @return primary key column name from annotation entity class
+     * @see Id
+     */
+    public static Field getIdField(Class<?> clazz) {

```

Comment:
Can we add a check on the `Entity` annotation?

Reply:
Added a check in https://github.com/Svydovets-Bobocode-Java-Ultimate-3-0/Bibernate/pull/5